src/main/java/com/example/pokebattlez/battle/model/Nature.java

```diff
@@ -0,0 +1,27 @@
+package com.example.pokebattlez.battle.model;
+
+import static com.example.pokebattlez.battle.model.Stat.*;
+
+public enum Nature {
```

Comment:
It would be great to have a JavaDoc here.  
What does it mean that _Hardy_ has ATTACK up and ATTACK down?

Reply:
Each nature affects 2 stats, one gets lowered by x1.1 and the other one gets reduced by x0.9